dockerfile/anaconda-ci/Dockerfile

```diff
@@ -58,7 +64,7 @@ RUN set -ex; \
   else \
     dnf copr enable -y ${copr_repo} fedora-eln-x86_64; \
   fi; \
-  curl -L https://raw.githubusercontent.com/rhinstaller/anaconda/${git_branch}/anaconda.spec.in | sed 's/@PACKAGE_VERSION@/0/; s/@PACKAGE_RELEASE@/0/; s/%{__python3}/python3/' > /tmp/anaconda.spec; \
```

Comment:
I would like to know why we used `curl` in the first place. Isn't there a use case that we are overlooking?

Reply:
tl;dr I don't think so, fortunately. The containers changed only incrementally since inception, and the use case is no longer valid.

The containers were originally written from the perspective of Github actions. That is, they would be built in a clean environment, with no context except for "for branch X". We feed all the important bits to them via arguments. These arguments bubble down from the action that has the actual config, via makefile, and only then to podman/docker. There was also the part that we would be building them daily, and reuse, so there was no context of PR, just the repo state. So straight `curl thing-from-repo` was good enough.

I also think that as we then went to use them on PRs, and needed to rebuild them if needed, and given that [back then] the tests would run always, I guess we decided to use as much from the repo as possible to minimize hackable surface. That is no longer true, all container builds directly on tests are behind the button to let them run, which is supposed to be clicked only after review of the changes.

Reply:
I wasn't able to find any use of the curl other than this one. I run our tests locally and builds locally and both worked as expected without the curl.